integration/testdata/java/java-with-vuln.yaml

```diff
@@ -1,10 +0,0 @@
-apiVersion: v1
-kind: Pod
-metadata:
-  name: java-with-vuln
-spec:
-  containers:
-  - name: java-with-vuln
-    image: gcr.io/kritis-int-test/java-with-vuln@sha256:b3f3eccfd27c9864312af3796067e7db28007a1566e1e042c5862eed3ff1b2c8
-    ports:
-    - containerPort: 80
```

Comment:
this file wasn't deleted just renamed (not sure why github isn't showing that)

Reply:
yeah strange.  it should show an rename and a change.